src/assets/css/librarybrowser.scss

```
@@ -738,18 +733,34 @@
     }
 }
 
+[dir=ltr] .mainDetailButtons {
+    @media all and (max-width: 62.5em) {
+        margin-left: -0.5em;
+    }
+}
+
+[dir=rtl] .mainDetailButtons {
+    @media all and (max-width: 62.5em) {
+        margin-right: -0.5em;
+    }
+}
```

Tbh, I don't understand the purpose of these margins.

To avoid duplicating `@media`:
```suggestion
@media all and (max-width: 62.5em) {
    [dir=ltr] .mainDetailButtons {
        margin-left: -0.5em;
    }

    [dir=rtl] .mainDetailButtons {
        margin-right: -0.5em;
    }
}
```

---

Me neither. I tried applying them but I really don't see a difference.

---

I removed them in my latest commit.